wolox-books/src/app/components/navbar/navbar.component.ts

```diff
@@ -0,0 +1,18 @@
+import { Component, OnInit } from '@angular/core';
+import { Observable } from 'rxjs';
+import { UserService } from 'src/app/services/user.service';
+
+@Component({
+  selector: 'app-navbar',
+  templateUrl: './navbar.component.html',
+  styleUrls: ['./navbar.component.scss'],
+})
+export class NavbarComponent implements OnInit {
+  isLoggedIn: Observable<boolean>;
+
+  constructor(public userService: UserService) {
+    this.isLoggedIn = userService.isLoggedIn();
```

Comment:
Put the logic of the component in ngOnInit method

Reply:
Done 😄